### PR TITLE
Features/houston event

### PR DESCRIPTION
**Pull request by zbrotherton**
One-score-load has to be merged before this can be reviewed

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -28,7 +28,7 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-  public static final boolean isCompetition = true;
+  public static final boolean isCompetition = false;
```
Comment (azvanderpas) [inline src/main/java/frc/robot/Robot.java]: Revert

File: src/main/java/frc/robot/classes/Auton.java [hunk 2]
```diff
@@ -184,10 +205,13 @@ private Command getAutonSequence(){
             case ONESCORE:
                 return
                     new SequentialCommandGroup(
+                        resetOdometry("1Score"),
+                        highTransitionSequenceCone(),
+                        translateY(-0.1, 0.5),
                         resetOdometry("1Score"),
                         highScoreSequenceCone(),
                         swerveSubsystem.followTrajectory("1Score", getTrajectory("1Score"))
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java:213]: I don't think I merged this correctly :/

File: src/main/java/frc/robot/classes/Auton.java [hunk 3]
```diff
@@ -226,29 +250,38 @@ private Command getAutonSequence(){
...
                     );
             case ONESCORECHARGE5:
                 return
                     new SequentialCommandGroup(
+                        resetOdometry("1ScoreCharge-5"),
+                        highTransitionSequenceCone(),
+                        translateY(-0.1, 0.5),
                         resetOdometry("1ScoreCharge-5"),
                         highScoreSequenceCone(),
                         swerveSubsystem.followTrajectory("1ScoreCharge-5", getTrajectory("1ScoreCharge-5"))
```
Comment (azvanderpas) [inline src/main/java/frc/robot/classes/Auton.java:282]: Don't think I merged this correctly either :/

**Review by azvanderpas — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/ArmSubsystem.java [hunk 4]
```diff
@@ -535,6 +543,8 @@ public void periodic() {
 
     actualXPositionLog.log(armXPosition);
     actualYPositionLog.log(armYPosition);
+    SmartDashboard.putNumber("Inverse Elbow", Units.radiansToDegrees(convertToElbowAngle(armXPosition, armYPosition)));
```
Comment (caseyjbrotherton) [inline src/main/java/frc/robot/subsystems/ArmSubsystem.java]: Why is this added?  Seems like we should have the inverse in the actualAngle, and that this is debugging code?

Comment (caseyjbrotherton) [reply src/main/java/frc/robot/subsystems/ArmSubsystem.java]: removed

**Review by caseyjbrotherton — APPROVED** [review]

**Review by azvanderpas — APPROVED** [review]
